src/PkgAuthentication.jl

```diff
@@ -263,47 +292,52 @@ function get_server_dir(url::AbstractString, server=Pkg.pkg_server())
     joinpath(Pkg.depots1(), "servers", domain)
 end
 
-function token_path(url)
+function token_path(url::AbstractString)
     if is_new_auth_mechanism()
         server_dir = get_server_dir(url)
         if server_dir !== nothing
             return joinpath(server_dir, "auth.toml")
         end
     end
-    get(ENV, "JULIA_PKG_TOKEN_PATH", joinpath(homedir(), ".julia", "token.toml"))
+    # older auth mechanism uses a different token location
+    # QUESTION: do we still need this for Julia ≥ 1.3?
```

Comment:
I think so, yes. `is_new_auth_mechanism` is only true for 1.4 onwards, IIRC.

Reply:
Ah, ok. So 1.3 is the cusp release that we're still supporting that does something different.